packages/cozy-konnector-libs/src/libs/saveBills.js

```diff
@@ -63,7 +65,11 @@ const requiredAttributes = {
  *
  * @alias module:saveBills
  */
-const saveBills = async (entries, fields, options = {}) => {
+const saveBills = async (inputEntries, fields, inputOptions = {}) => {
+  // Cloning objects
```

Comment:
What do you think of 

```
// Cloning input arguments since both entries and options are expected
// to be modified by functions called inside saveBills.
```

Reply:
Much better, I added it.